.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225369382-f19130f7-4a7b-4d07-92ca-a47a369fd903.png)

- [ ] It would be nice if you remove the section descriptions from both Authors and Future Feature section to make your readme file descriptive and clean 😊 These are just instructions for the very first time when you add this readme file 👍 After inserting your data in your readme file you are supposed to delete all the instructions from your readme file to make it clean and professional 🤔